### ISSUE TITLE
Why does it claim to need Conda, when it appears to have CMake support?

### ISSUE THREAD

**Issue by barracuda156** [lcompilers/lpython#2446]
README lists Conda as a requirement: https://github.com/lcompilers/lpython/blob/0a79bf55d34a35d1d733e376c9c85301ae7b5b3f/README.md?plain=1#L28-L32
Why is that needed at all?

**Comment by certik** [comment]
LPython doesn't need conda, only the dependencies. We only put Conda there to have a complete list of instructions to getting it up and running, but if you install the dependencies some other way (say using `apt-get` in Debian), then it will work as well.

Do you want to submit a PR clarifying this?

**Comment by barracuda156** [comment]
@certik I can try building it, if that works, then yes.

Will it be useful by the way without LLVM? I see CMakeLists has it as optional, so provisionally it may, or otherwise a claim re working across all systems fail, since unlike GCC all LLVM stuff is not properly portable and is known to be broken on a number of platforms.

**Comment by certik** [comment]
Yes, LPython works without LLVM, you can use all the other backends instead. For example at https://dev.lfortran.org/ we use LPython WASM build without LLVM, and we use our WASM backend.

**Comment by barracuda156** [comment]
@certik I finally got back to this. There are two trivial fixes needed:
https://github.com/lcompilers/lpython/pull/2459
https://github.com/lcompilers/lpython/pull/2461

And one issue to look at, if you could: https://github.com/lcompilers/lpython/issues/2460
